assets/src/js/global.js

```diff
@@ -1,3 +1,19 @@
+/* eslint-env es6 */
+/* eslint-disable */
+
 /**
  * Generic site JavaScript.
  */
+
+/**
+ * Search for Donation Button and add parameter link Op1.
+ */
+document.querySelectorAll('[href^="https://donate.wikimedia.org"]').forEach( function( item ) {
+	'use strict';
+	var page_id = window.post_id;
+	var url = item.href;
+	var params = new URLSearchParams( url.search );
+	params.set( 'utm_source', page_id );
+	item.href = url.replace( /\?.*/, '?utm_source=' + page_id );
```

Comment:
Probably want 
```suggestion
	item.href = url.replace( /\?.*/, params.toString() );
```
here, to not lose other parameters

Reply:
Updated in [9bb40a2](https://github.com/wikimedia/shiro-wordpress-theme/pull/17/commits/9bb40a25747d9e40149eb8bd5825aeafccda3636).